src/LintingViewExtension/LintingViewExtension.cs

```diff
@@ -39,6 +39,8 @@ public override void Loaded(ViewLoadedParams viewLoadedParams)
             this.linterViewModel = new LinterViewModel(linterManager, viewLoadedParamsReference);
             this.linterView = new LinterView() { DataContext = linterViewModel };
 
+            linterViewModel.ActiveLinter = linterManager.ActiveLinter;
```

Comment:
we can remove this if we rely on LinterManager direct access for ActiveLinter property